docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -311,13 +311,54 @@ Returns language translation keys as JSON string.
 
 ### [{{getContentImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImage.js)
 

```

Comment:
I am curious if the links in the Helper column should also go to github pages like getContentImage. I am wondering why the links are needed in this column.

Reply:
This isn't related to the fix nate requested, so it should go in a separate ticket and not hold up this pull request.